develop/buildkite/pipeline.yml

```diff
@@ -306,7 +325,8 @@ steps:
           run: integration-test-sqlite
           config: ./develop/buildkite/docker-compose.yml
 
-  - wait
+  - wait: ~
+    continue_on_failure: true  # required for test reports of failed build to be picked up
```

Comment:
Note that - as expected - the coverage-report step will fail if any of the previous steps failed. That's because it uses `develop/buildkite/scripts/coverage-report.sh` to download the coverage data from each step. If only one is missing, the entire step is aborted. This means we won't get partial coverage data uploaded for failed builds.

Reply:
There's no way to query buildkite to get the status of the steps run so far?

Or what if we just have it count the files it got?

Reply:
There is a way to query the outcome of a single step, but I couldn't find any way to query the status of the entire build. But I found a way now by using an additional `wait` step.